fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -22,6 +22,15 @@ def take_off
           print_bundle_exec_warning(is_slow: (Time.now - before_import_time > 3))
         end
 
+        unless ENV['LANG'].end_with?("UTF-8") || ENV['LC_ALL'].end_with?("UTF-8")
```

Comment:
Does this compare need to be case insensitive? 🤔

Reply:
I'm not sure but I think it is better to err on the side of caution here: theoretically your locale has to match one of the outputs of `locale -a`, which use the uppercased `UTF-8` spelling